systemtest/src/test/java/io/strimzi/systemtest/tracing/TracingAbstractST.java

```diff
@@ -78,10 +78,12 @@ public abstract class TracingAbstractST extends AbstractST {
 
     private static final Logger LOGGER = LogManager.getLogger(TracingAbstractST.class);
 
-    private Stack<String> jaegerConfigs = new Stack<>();
+    private String jaegerConfigs;
+    private File certManager;
```

Comment:
You can `create`or `apply` directly based on String path, any specific reason for this? I don't see any changes done in the file so for deletion you can use the same file as well?

Reply:
Good point, `certManager` is not necessary there.
`jaegerConfigs` contains modified file, so instead of "creating" it again in deletion method, with all the replacements, I thought it will be simpler to store it like this.

Reply:
Removed the `certManager` variable